control-plane/api-gateway/controllers/gateway_controller.go

```diff
@@ -57,7 +59,7 @@ type GatewayController struct {
 // Reconcile handles the reconciliation loop for Gateway objects.
 func (r *GatewayController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := r.Log.WithValues("gateway", req.NamespacedName)
-	log.Info("Reconciling the Gateway: ", req.Name)
+	log.Info("Reconciling the Gateway: ")
```

Comment:
I did this initially and it was causing errors, will update the message to match other controllers